trinity/main.py

```diff
@@ -196,6 +199,17 @@ def main() -> None:
         )
 
 
+def config_noisy_logger(args: Namespace) -> None:
+    # These loggers generates too much DEBUG noise, drowning out the important things
+    # We will use the log_level if it's set, otherwise default to INFO
+    if args.log_levels:
+        logging.getLogger('p2p.kademlia').setLevel(args.log_level)
+        logging.getLogger('p2p.discovery').setLevel(args.log_level)
+    else:
+        logging.getLogger('p2p.kademlia').setLevel(logging.INFO)
+        logging.getLogger('p2p.discovery').setLevel(logging.INFO)
```

Comment:
@glaksmono I think we can do this more generic by creating a constant `LOG_LEVEL_DEFAULTS` which is basically a `Dict[str, int]` to provide the default log levels and then merge those with the `log_level` that the user may provide via the command line (where the command line arguments take precedence over the defaults).